### PR TITLE
Add support for postgreql to standard-application-stack chart

### PR DESCRIPTION
**Pull request by barrymars**

### CODE REVIEW

File: charts/standard-application-stack/templates/secrets.yaml [hunk 1]
```diff
@@ -194,3 +194,58 @@ data:
...
+  labels: {{ include "mintel_common.labels" . | nindent 4 }}
+  annotations: {{ include "mintel_common.commonAnnotations" . | nindent 4 }}
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  DB_HOST: {{ printf "%s-postgresql" (include "mintel_common.fullname" .) | b64enc }}
+  DB_NAME: {{ .Values.postgresql.auth.database | b64enc }}
+  DB_USER: {{ .Values.postgresql.auth.username | b64enc }}
+  DB_PASSWORD: {{ .Values.postgresql.auth.password | b64enc }}
+  postgresql-postgres-password: {{ .Values.postgresql.auth.password | b64enc }}
```
Comment (nabadger) [inline charts/standard-application-stack/templates/secrets.yaml:241]: What's this one for? and the one below?

Comment (barrymars) [reply charts/standard-application-stack/templates/secrets.yaml:241]: it's becauase it's used in several places we don't control
postgresql-* are used by the helm chart / containers while provisioning the database
PG* are used by postgresclient
DATA_SOURCE_* used by postgres exporter

actually some of them need adding to the ExternalSecret too

Comment (nabadger) [reply charts/standard-application-stack/templates/secrets.yaml:241]: Ok, plus it's only for `local` too which is cool.

So do we need the `PG*` and `DATA_SOURC*` ones in the non-local cluster too?

Comment (barrymars) [reply charts/standard-application-stack/templates/secrets.yaml:241]: yeah, as that's the variables the tools are expecting (postgres clearly doesn't like consistency as much as mysql/mariadb) :)

**Review by nabadger — APPROVED** [review]

**Pull request merged**